src/test/java/pl/eukon05/eventboard/event/application/service/UnitTests.java

```diff
@@ -140,4 +144,38 @@ void should_attend_invited_event() {
 
         assertTrue(attendEventUseCase.execute(userID, event.getId()));
     }
+
+    @Test
+    void should_invite_to_public_event() {
+        Event event = createTestEvent();
+        String friendID = "somerandomid";
+        Mockito.when(getEventPort.getEventById(Mockito.anyLong())).thenReturn(Optional.of(event));
+        Mockito.when(getUserFriendlistPort.getUserFriendlist(Mockito.anyString())).thenReturn(List.of(friendID));
+
+        assertTrue(inviteToEventUseCase.execute(userID, friendID, event.getId()));
+        assertTrue(event.getInviteeIDs().contains(friendID));
+        verify(getUserFriendlistPort).getUserFriendlist(userID);
+        verify(getEventPort).getEventById(event.getId());
+        verify(saveEventPort).saveEvent(event);
+    }
+
+    @Test
+    void should_invite_to_private_event() {
+        Event event = createTestEvent();
+
+        event.setType(EventType.PRIVATE);
+        event.setOrganizerID(userID);
+
+        String friendID = "somerandomid";
+        Mockito.when(getEventPort.getEventById(Mockito.anyLong())).thenReturn(Optional.of(event));
+        Mockito.when(getUserFriendlistPort.getUserFriendlist(Mockito.anyString())).thenReturn(List.of(friendID));
+
+        assertTrue(inviteToEventUseCase.execute(userID, friendID, event.getId()));
```

Comment:
I would move `when` section (`inviteToEventUseCase.execute`) out of `then`.
```suggestion
        var invitiationResult = inviteToEventUseCase.execute(userID, friendID, event.getId());
        
        assertTrue(invitiationResult)
```